templates/engage/shift-to-app-store-experience.html

```diff
@@ -0,0 +1,82 @@
+{% extends_with_args "engage/base_engage.html" with title="Linux app stores provide improved application visibility and usage" meta_image="https://assets.ubuntu.com/v1/8b9a82bd-Embedded+social+media+banner.jpg" meta_description="Linux app stores are successfully alleviating historical challenges faced by software developers whilst improving visibility to enterprises and end users." %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1MsQpa3ik8w-jVAmPdsQIz0w_XteBsH-g2xDVigPeo6Q{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-takeover">
+  <div class="p-strip--image is-dark is-deep p-takeover__suru">
+    <div class="row u-equal-height">
+      <div class="col-7 u-vertically-center">
```

Comment:
If you remove the `u-vertically-center` class here you can remove the extra div below this line.

Reply:
oh nice - ta